src/style-stars--0.css

```diff
@@ -0,0 +1,3 @@
+.stars--0 .stars__star {
+  margin-top: 0;
+}
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Instead of styling `.stars--0 .stars__star`, you should use a class to style these elements. This will make your stylesheet more maintainable and your selectors more efficient.